aries-site/src/components/feedback/Feedback.js

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  Heading,
+  Layer,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const Feedback = ({
+  children,
+  modal,
+  onChange,
+  onClose,
+  onEsc,
+  onSubmit,
+  onReset,
+  show,
+  title,
+  value,
+  isSucessful,
+}) => {
+  const size = useContext(ResponsiveContext);
+
+  let content = (
+    <Box
+      fill="vertical"
+      overflow="auto"
+      width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+      pad="medium"
+      flex
+    >
+      <Identifier onClick={onClose} title={title} modal={modal} />
+      <Box gap="medium" margin={{ vertical: 'small' }}>
+        <Form
+          value={value}
+          onChange={onChange}
+          onSubmit={onSubmit}
+          onReset={onReset}
+          method="post"
+          validate="submit"
+        >
+          {children}
+          <>
+            {isSucessful !== true ? (
+              <Box
+                margin={{ top: 'medium', bottom: 'small' }}
+                justify="end"
+                gap="medium"
+                direction="row"
+              >
+                <Button onClick={onClose} label="Cancel" />
+                <Button
+                  onSubmit={onSubmit}
+                  label="Submit"
+                  primary
+                  type="submit"
+                />
+              </Box>
+            ) : (
+              <Box align="end" margin={{ top: 'medium', bottom: 'small' }}>
```

Comment:
```suggestion
              <Box align="end" pad={{ top: 'medium', bottom: 'small' }}>
```

Reply:
good point I actually just did this in forms we should probably change that to 😕